Project-Euler/Project-Euler/Problems.cs

```diff
@@ -23,12 +23,36 @@ private static void Problem1()
                 }
             }
 
-            Console.WriteLine($"Answer: {sum}");
+            Console.WriteLine($"\n\tSolution: {sum}");
         }
 
-        private static void Problem2()
+        public static void Problem2()
         {
-            Console.WriteLine("Problem: By considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms.");
+            Console.WriteLine("\n\t2) By considering the terms in the Fibonacci sequence whose values do not exceed four million, find the sum of the even-valued terms.");
+
+            int firstNum = 1;
+            int secondNum = 2;
+            int thirdNum;
```

Comment:
I would try to improve the naming for these three variables. 
Maybe something like:

- previousFibonacciNumber
- currentFibonacciNumber
- nextFibonacciNumber